bikeshare_app/views.py

```diff
@@ -1,26 +1,73 @@
 """Define the views for the app"""
 from flask import render_template
-from bikeshare_app import app
-from bikeshare_app.models import ActiveShare
+from flask import request
+from flask import redirect
+from flask import url_for
+from flask import flash
+import flask.ext.login as flask_login
+from . import app
+from . import login_manager
+from .models import ActiveShare
+from os import getenv
+
+
+class User(flask_login.UserMixin):
+    """User model for login."""
+    pass
+
+
+@login_manager.user_loader
+def user_loader(email):
+    """Load a user from a session."""
+    # TODO: Hookup email verification to DB, verify they exist
```

Comment:
Although for now we should probably just leave the `TODO` since we have no way to add users....